internal/collect/clusters.go

```diff
@@ -5,23 +5,28 @@ package collect
 
 import (
 	"context"
-
+	"github.com/newrelic/nri-vsphere/internal/config"
 	"github.com/newrelic/nri-vsphere/internal/performance"
 	"github.com/vmware/govmomi/vim25/types"
 
-	"github.com/newrelic/nri-vsphere/internal/load"
 	"github.com/vmware/govmomi/vim25/mo"
 )
 
 // Clusters VMWare
-func Clusters(config *load.Config) {
+func Clusters(config *config.Config) {
 	ctx := context.Background()
 	m := config.ViewManager
 
+	collectTags := config.TagCollectionEnabled()
```

Comment:
why do we need these two variables, isn't the value used just once? Is it to improve readability?

Reply:
you're right they are not needed.